packages/regl-worldview/src/Worldview.js

```diff
@@ -25,10 +25,10 @@ export type BaseProps = {|
   showDebug?: boolean,
   children?: React.Node,
 
-  defaultCameraState: CameraState,
-  // Worldview is controlled if cameraState and onCameraStateChange are both present
+  // Worldview is only controlled if both cameraState and onCameraStateChange are present
```

Comment:
I think it should be considered controlled if `cameraState` is present, and uncontrolled if it absent. The presence of `onCameraStateChange` is not relevant other than to indicate whether the developer cares about callbacks when the user tries to move the camera. Theoretically you could have an uncontrolled component with an onChange handler, or a controlled component without one.

I guess for normal React form fields, you usually wouldn't set `value` without `onChange` or `onChange` without `value`, or `onChange` with `defaultValue`, but for Worldview I think those are ok. What do others think?

Reply:
My interpretation of `controlled` was that Worldview will always take in the newest `cameraState` from props. But it's not the case now since we only update internal `cameraState` when both `cameraState` prop and `onCameraStateChange` props are present. I'm considering:

1. Remove the comment here to avoid confusion since the console msgs and docs are very explicit about what are needed for controlled/uncontrolled props
2. Always update internal `cameraState` if the prop exists since it's closer to what native input box would do. Currently it only updates if both `cameraState` and `onCameraStateChange` are present. 

What do you think?

Reply:
2nd option sounds good, the current behavior you mentioned sounds confusing.

Reply:
Agreed with 2nd option. To compare with, say, `<input>` field: passing in `value` but not `onChange` makes the text box read-only. We should probably behave in the same way.